File: androidchallenge1/src/main/java/class28/androidchallenge01/ActSplash.java
package class28.androidchallenge01;

import android.content.Intent;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import class28.androidchallenge01.ui.login.ActLoginRegister;

public class ActSplash extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        startActivity(new Intent(this, ActLoginRegister.class));
        finish();
    }
}

File: androidchallenge1kotlin/src/main/java/class28/androidchallenge01kotlin/ActSplash.kt
package class28.androidchallenge01kotlin

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import class28.androidchallenge01kotlin.ui.login.ActLoginRegister

class ActSplash : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        startActivity(Intent(this, ActLoginRegister::class.java))
        finish()
    }
}